robot-server-plugin/src/main/kotlin/com/intellij/remoterobot/RobotServerImpl.kt

```diff
@@ -259,7 +259,11 @@ class RobotServerImpl {
                 get("/{componentId}/screenshot") {
                     call.dataResultRequest({
                         val componentId = call.parameters["componentId"] ?: throw IllegalArgumentException("empty componentId")
-                        ideRobot.makeScreenshot(componentId)
+                        val paintingMode = call.parameters["withPainting"].toBoolean()
```

Comment:
I would name the property `isPaintingMode`

Reply:
renamed